lang/cachyos-kernel-manager_tr.ts

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="tr">
+<context>
+    <name>ConfOptionsPage</name>
+    <message>
+        <source>Enable CachyOS config</source>
+        <translation>CachyOS yapı biçimleyiciyi etkinleştir</translation>
+    </message>
+    <message>
+        <source>Tweak kernel options prior to a build via nconfig</source>
+        <translation>nconfig aracılığıyla yapılandırmadan önce çekirdek ayarlarını iyileştir</translation>
```

Comment:
"ayarlarına ince ayar yap" instead of "ayarlarını iyileştir"  is more suitable for this context in my opinion. Same goes with other translation where "iyileştir" was used.

Reply:
'ince ayar yapma' is more equivalent of 'to tune' or 'finely tune' than 'to optimise', so I disagree.